blocks/init/src/Blocks/assets/styles/application-blocks.scss

```diff
@@ -12,7 +12,7 @@
 @import './../../../../assets/styles/parts/shared';
 
 // Register Wrapper block styles.
-@import './../../wrapper/wrapper-style.scss';
+// @import './../../wrapper/wrapper-style.scss';
```

Comment:
If this is not used, can't it be removed?

Reply:
no this is a placeholder for future usage